nss-evening-cohort-14/gitsub-team-no-name
# Bj repositories

Finished all of my MVP requirements for the Repos page.

Description
Created a function that creates the form for the repos.
Created a function that prints those created repos to the DOM.

Related Issue
#15

Motivation and Context
This change is the finished MVP for Repo functionality.

How Can This Be Tested?
Filled out the form on the repo page. If it prints above the form with the Name and Description of the repo, it is working.
Make sure that the form resets after the repo is created.

Screenshots (if appropriate):
Types of changes
 Bug fix (non-breaking change which fixes an issue)
[x ] New feature (non-breaking change which adds functionality)
 Breaking change (fix or feature that would cause existing functionality to change)
